monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -209,7 +287,7 @@ def get_all_case_stats(self, key="training", transform_list=None):
         keys = list(filter(None, [self.image_key, self.label_key]))
         if transform_list is None:
             transform_list = [
-                LoadImaged(keys=keys, ensure_channel_first=True, image_only=True),
+                LoadImaged(keys=keys, ensure_channel_first=True, image_only=False),
```

Comment:
why do we need image_only=False? it seems it's an updated way to get meta dict

Reply:
yes, I need the meta_dict to print out which file failed.

Reply:
you should be able to use `batch_data.meta.get("filename_or_obj")`, please help change and test it...